Wavefront.CSharp.SDK/Common/Constants.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Wavefront.CSharp.SDK.Common
+{
+    /// <summary>
+    /// Class to define all sdk constants
+    /// </summary>
+    public static class Constants
+    {
+        /// <summary>
+        /// Use this format to send metric data to Wavefront.
+        /// </summary>
+        public static readonly string WavefrontMetricFormat = "wavefront";
```

Comment:
`public const string WavefrontMetricFormat` faster as it gets compiled down into a literal within the code; however, does have implications in how the clients consume our library (in that they would have to rebuild their clients with our version if we changed the value of our const).  If the value is never expected to change this may not be an issue.